src/components/Person/Person.jsx

```diff
@@ -1 +1,35 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+  let whoIsPartner = '';
+
+  if (isMarried && sex === 'm') {
+    whoIsPartner = `${partnerName} is my wife`;
+  } else if (isMarried && sex === 'f') {
+    whoIsPartner = `${partnerName} is my husband`;
+  } else {
+    whoIsPartner = 'I am not married';
+  }
```

Comment:
not so good to make this logic directly in component body, better to make it inside funk and call it when you need